app/views/admin/facility_groups/_form.html.erb

```diff
@@ -56,7 +55,7 @@
                     <i class="fas fa-trash-alt text-grey"
                        data-toggle="tooltip"
                        data-placement="right"
-                       title="Contains <%= block.facility_regions.count %> facility(s)"></i>
+                       title="Cannot delete blocks with facilities"></i>
```

Comment:
@timcheadle "blocks" should be localized. The title should be "Cannot delete ${blocks} with facilities".

Reply:
Fixing by changing the copy.